rest_framework-stubs/response.pyi

```diff
@@ -23,3 +27,12 @@ class Response(SimpleTemplateResponse):
     def render(self) -> Any: ...
     @property
     def status_text(self) -> str: ...
+
+class _MonkeyPatchedResponse(Response):
+    client: DjangoClient
```

Comment:
This can be `APIClient` here

Reply:
I wondered about that. What prevents a circular dependency?

Reply:
I believe circular dependencies are allowed in `.pyi` stubs files.

Reply:
Quite right. TIL